docs/codeql/codeql-language-guides/codeql-for-java.rst

```diff
@@ -1,9 +1,17 @@
 .. _codeql-for-java:
 
-CodeQL for Java
-===============
+CodeQL for Java and Kotlin
+==========================
 
-Experiment and learn how to write effective and efficient queries for CodeQL databases generated from Java codebases.
+Experiment and learn how to write effective and efficient queries for CodeQL databases generated from Java and Kotlin codebases.
+
+.. include:: ../reusables/kotlin-beta-note.rst
+
+.. include:: ../reusables/kotlin-java-differences.rst
```

Comment:
This note ended up longer than I expected. I'm wondering if it's needed on this page 🤔 

If we're going to include it, I'd tend to put both it and the note about enabling Kotlin support after the list of articles.

Reply:
I'm also wondering if including it here is somehow causing the heading to appear in the TOC for the parent language guides article:
<img width="970" alt="image" src="https://user-images.githubusercontent.com/1877141/203514303-3f26e08e-2749-4b55-8e98-913ec4acd6d5.png">

I haven't seen this behavior before and my Sphinx knowledge is rather rusty. Happy to experiment locally or to look at this together.

Reply:
Hmm, I'm not sure. Maybe it'd be easier to just take it out, per your first comment? 🤔

Reply:
Sounds like a good plan